pkgs/tools/misc/topgrade/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, rustPlatform }:
+{ stdenv, fetchFromGitHub, rustPlatform, darwin }:
```

Comment:
you should request `Foundation` instead of `darwin`. Here's an example:
- https://github.com/NixOS/nixpkgs/blob/d8d882e9978cff80e7ff8828203ebd419261c456/pkgs/top-level/all-packages.nix#L19044-L19047
- https://github.com/NixOS/nixpkgs/blob/d8d882e9978cff80e7ff8828203ebd419261c456/pkgs/applications/misc/kitty/default.nix#L8

Reply:
Should be resolved now. Thanks for the detailed correction and your work as a reviewer :thumbsup: